FE/src/App.tsx

```diff
@@ -1,26 +1,29 @@
 import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
 import { ThemeProvider } from 'styled-components';
+import { RecoilRoot } from 'recoil';
 import { GlobalStyle } from '@/styles/GlobalStyle';
 import { LoginPage } from '@/pages/Login';
 import { theme } from '@/styles/theme';
-import { Callback } from '@/pages/Callback';
+import { Callback } from '@/pages/LoginRedirect';
```

Comment:
이름 일관성 있게 룰을 맞춰서  가져가면 더 좋을 것 같습니다.
```suggestion
import { LoginRedirectPage } from '@/pages/LoginRedirect';
```
+  lazyload 코드스플리팅도 한 번 살펴보시면 좋을 것 같네요 
https://reactjs.org/docs/code-splitting.html#route-based-code-splitting